src/strand.rs

```diff
@@ -1,7 +1,8 @@
-#[derive(Debug, PartialEq, Clone, Copy)]
+#[derive(Debug, Clone, Copy, PartialEq)]
 pub enum Strand {
     Positive,
-    Negative
+    Negative,
+    Unknown
```

Comment:
I think we must not include `Unknown`. I do not think it ever happens that we have a "stranded" thing but we do not know what strand it is on.

Pls let me know if you know about any such examples, we can, of course, reconsider.